src/mlstacks/terraform/k3d-modular/variables.tf

```diff
@@ -7,6 +7,10 @@ variable "enable_orchestrator_kubernetes" {
   description = "Enable Kubernetes deployment"
   default     = false
 }
+variable "enable_orchestrator_skypilot" {
+  description = "Enable SkyPilot orchestrator deployment"
+  default     = false
+}
```

Comment:
Isn't skypilot cloud only? How would it relate to k3d-modular?

Reply:
Good point.